### ISSUE TITLE
[SMOKE TESTING] VitaGrafix v3.1 Preview Build

### ISSUE THREAD

**Issue by Shinriji** [Electry/VitaGrafix#102]
Opened to more people crash test the 2.5 preview to check for issues before release, since the original post was buried elsewhere. Please only post test results of the preview build on this topic, not game requests.

Originally posted by Electry

[Test build: [VitaGrafix_testbuild_v2.5.zip](https://github.com/Electry/VitaGrafix/files/2363789/VitaGrafix_testbuild_v2.5.zip)

It has:

Sly Cooper 1, 2, 3 FB + FPS patch (PCSF00269, PCSF00270, PCSF00271, PCSF00338, PCSA00095, PCSA00096, PCSA00097, PCSA00098)
Sly Cooper 4 FPS patch (PCSF00156, PCSF00206, PCSF00207, PCSF00208, PCSF00209, PCSA00068)
Ratchet & Clank 2 FB patch (PCSF00482, PCSF00485, PCSA00133)
Ninja Gaiden Sigma 2 Plus IB patch (PCSB00294, PCSE00233, PCSG00157)

**Comment by houlaiqingtian** [comment]
thanks

**Comment by playerlilin** [comment]
This will helps a lot.Thanks for hard working.

**Comment by Shinriji** [comment]
I tested with Sly 4 (US cart version, 500mhz OC) for almost 2 hours and everything went as expected.

**Comment by Shinriji** [comment]
Gonna quote some more testing results of the 2.5 preview:

Originally posted by ChiSys

> Have played now from the first planet to the third planet in a row and didn't found something game breaking in Ratchet 2. Appart from some UI masking issues in the menues the game runs pretty sweet and looks really nice now.
> Little hint: Ingame hint messages can be disabled and if so you doesn't notice the issues while playing. Appart from the menues but they are all working and useable.
> For me its acceptable and I hope you will look into 1 & 3 also.
> 
> Ninja Gaiden 2 works for me and I haven't noticed something is broken. Only loading times are feeling longer but I can be wrong.
> 
> Sly games working like a charm.

**Comment by SheidCraft** [comment]
in Ninja Gaiden 2 there are still problems with fps in some places. And it's very noticeable. In principle, given that the original was with a dynamic resolution (480p max), this is fixable. Making it dynamic again, but setting 544 as maximum

**Comment by Shinriji** [comment]
> in Ninja Gaiden 2 there are still problems with fps in some places. And it's very noticeable. In principle, given that the original was with a dynamic resolution (480p max), this is fixable. Making it dynamic again, but setting 544 as maximum

This plugin won't fix FPS issues on games and most of the time there will be tradeoffs, but yeah, options are good (if possible).

**Comment by SheidCraft** [comment]
I did not say that this plugin should raise fps. I said that in Ninja Gaiden at 544p there are falls. Even at a frequency of 500 MHz

**Comment by Electry** [comment]
[VitaGrafix_v3.0-alpha.zip](https://github.com/Electry/VitaGrafix/files/2474364/VitaGrafix_v3.0-alpha.zip)

Now uses ioPlus v0.1 instead of kuio. Add ioPlus.skprx to *KERNEL section of taihen config.txt and remove kuio.skprx if none of your other plugins use it.
All game options are disabled/OFF by default, so you need to add those you want to use to VitaGrafix config.

It has everything v2.5 had + added Ninja Gaiden Sigma 2 Plus dynamic scaling back (as requested), so now you need to specify two IB resolutions (like this: "IB=960x544,720x408") for that game.

Hopefully nothing broke during the rewrite. Updated README.md is available in *[experimental](https://github.com/Electry/VitaGrafix/tree/experimental)* branch.

**Comment by kofwinner** [comment]
I suggest all game options are enabled/ON by default

**Comment by Shinriji** [comment]
Great. Gonna download and test it later today.

**Comment by Electry** [comment]
@Shinriji @Wontell yeah, but some games suffer from performance issues or bugs at 960x544 and I didn't want to force that to everyone (by default).

@TheMostEve yep

any issues with the latest build?

**Comment by Wontell** [comment]
> @Shinriji @Wontell yeah, but some games suffer from performance issues or bugs at 960x544 and I didn't want to force that to everyone (by default).

Its much easier to disable one game then to enable full list of them.

Q: I somewhat got confused, all the games with 704x448, 720x408, 840x476 resolution that [says Default resolution in your readme] is this their native resolution and they can get higher up to 960x544
or its your tweaked maximum resolution possible by the plugin?
If im not wrong Your previous plugins automatically started all the game in their highest resolution, 960x544 [if game allowed]

**Comment by Electry** [comment]
[VitaGrafix_v3.1-alpha.zip](https://github.com/Electry/VitaGrafix/files/2476347/VitaGrafix_v3.1-alpha.zip)

What's new:

Dragon Quest Builders (EU/US/JP/ASIA) FPS cap patch (now you can force 30 fps lock)
Ratchet & Clank [EU/US] FB res. patch
Ratchet & Clank 3 [EU/US] FB res. patch (this one has the same clipping issues as R&C 2)
Ratchet & Clank: Full Frontal Assault / QForce [EU/US] FB res. patch
Borderlands 2 (PCSF00576) FB res. patch (thanks to vicadinsx)
Utawarerumono: Chiriyuku Mono he no Komoriuta [JP] IB res. patch
LEGO Star Wars: The Force Awakens [EU/US] 960x544 GPU crashes hopefully resolved
Dragon Ball Z: Battle of Z [EU/US/JP] IB res. patch

Again, I did not test those games extensively. Please let me know of any issues you encounter.
Updated README.md is in the *[experimental](https://github.com/Electry/VitaGrafix/tree/experimental)* branch.

**Comment by Shinriji** [comment]
Downloaded the latest version and vitagrafix configurator. I have Dragon Ball, so I can test it ASAP.

**Comment by SheidCraft** [comment]
Whats me doing? Version 3.1 does not work if Kuio is installed, but if I remove Kuio, Vita it will hang on the ENSO iconWhat do i do? Version 3.1 does not work if Kuio is installed, but if I remove Kuio Wita it will hang on the ENSO icon

**Comment by Shinriji** [comment]
I tested DBZ Battle of Z (US) for about 1 hour and there is no issues with the game. Framerate was a stable 30FPS with 960x544 (500mhz OC), but I didn't seen a great improvement though.

**Comment by Shinriji** [comment]
I tested DBZ Battle of Z (US) for about 1 hour and there is no issues with the game. Framerate was a stable 30FPS with 960x544 (500mhz OC), but I didn't seen a great improvement though.

**Comment by kofwinner** [comment]
I suggest all game options are enabled/ON by default again.
Or is there a section to enable all game modifications in VitaGrafix configuration file？

**Comment by Shinriji** [comment]
I saw that the experimental branch have new commits (3.2?), maybe you can create a suprx so we can test it out?

**Comment by SheidCraft** [comment]
@Shinriji 3.2? Link please...

**Comment by Shinriji** [comment]
> @Shinriji 3.2? Link please...

That's why I asked if Electry compiled a new suprx for testing. No links for now and maybe he want to add more stuff before creating a new preview build.
